### PR TITLE
initial version

### PR DESCRIPTION
**Pull request by anusha-reflektion**

### CODE REVIEW

File: Reflektion.tpl [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+    ]
+  },
+  {
+    "displayName": "Event - Add To Cart",
+    "name": "a2c",
+    "groupStyle": "ZIPPY_CLOSED",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "help": "pdp OR home OR category OR cart OR qview",
```
Comment (ManishaKundoor-Reflektion) [inline Reflektion.tpl]: typo at view

File: Reflektion.tpl [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+    "subParams": [
+      {
+        "help": "Comma separated rfkids. Products Context will be applied to all Reflektion widgets in the page, if nothing is specified. Example: rfkid_1,rfkid_2",
+        "displayName": "rfkids",
+        "simpleValueType": true,
+        "name": "rfkids",
+        "type": "TEXT"
+      },
+      {
+        "help": "Create a variable, populate it with list of SKUs in the cart page and use the variable here",
```
Comment (ManishaKundoor-Reflektion) [inline Reflektion.tpl]: remove the text 'cart page'

File: Reflektion.tpl [hunk 3]
```diff
@@ -0,0 +1,503 @@
...
+    ]
+  },
+  {
+    "displayName": "Event - Order Confirm",
+    "name": "orderConfirm",
+    "groupStyle": "ZIPPY_CLOSED",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "help": "List of Product Objects. See Reflektion documentation for details",
```
Comment (ManishaKundoor-Reflektion) [inline Reflektion.tpl]: Is it events api doc?

File: Reflektion.tpl [hunk 4]
```diff
@@ -0,0 +1,503 @@
...
+// Enter your template code here.
+var log = require('logToConsole');
+
+log('data =', data);
+
+function getProductList(products, returnSkuList) {
+  var additionalKeys = [
+        {name: 'price', type: 'number'}, 
+        {name: 'quantity', type: 'number'}, 
+        {name: 'psku', type: 'string'},
```
Comment (ManishaKundoor-Reflektion) [inline Reflektion.tpl]: is it intended to be `psku` or is it a typo?

File: Reflektion.tpl [hunk 5]
```diff
@@ -0,0 +1,503 @@
...
+      });
+    }
+  }
+  return productList;
+}
+
+function getUserDetails(userDetails) {
+  var userObject;
+  if (userDetails) {
+  	if (typeof userDetails === 'string') {
```
Comment (ManishaKundoor-Reflektion) [inline Reflektion.tpl]: indentation

File: Reflektion.tpl (hunk 1 above)
Comment (anusha-reflektion) [reply Reflektion.tpl]: it is qview in the API doc (for quick view)

File: Reflektion.tpl (hunk 4 above)
Comment (anusha-reflektion) [reply Reflektion.tpl]: psku for **parent SKU** :)

File: Reflektion.tpl (hunk 5 above)
Comment (anusha-reflektion) [reply Reflektion.tpl]: fixed indentation

**Pull request merged**